lib/crypto/eosdart_ecc/src/key.dart

```diff
@@ -21,8 +21,7 @@ class EOSPublicKey extends EOSKey {
 
   /// Construct EOS public key from string
   factory EOSPublicKey.fromString(String keyStr) {
```

Comment:
These changes are from auto save applying our formatting rules. No changes to functionality.